tests/hmm/hmumu_utils.py

```diff
@@ -1440,6 +1445,8 @@ def get_selected_muons(
             mu_phi = NUMPY_LIB.array(mu_phi)
             mu_mass = NUMPY_LIB.array(mu_mass)
             mu_iso = NUMPY_LIB.array(mu_iso)
+
+        is_not_fsr = (muons.pt == mu_pt)
```

Comment:
I got an error message when running this:

in get_selected_muons
    "is_not_fsr": is_not_fsr,
UnboundLocalError: local variable 'is_not_fsr' referenced before assignment

Reply:
Yes, that was a bug when not using the option `do_fsr`. It's fixed now.